striot-gui.cabal

```
@@ -69,10 +67,9 @@ test-suite striot-gui-test
   build-depends:
       aeson >=1.4.6.0
     , algebraic-graphs >=0.1
-    , base >=4.7 && <5
-    , bytestring >=0.10.10.0
+    , base >=4.9
+    , bytestring >=0.9.2
+    , hspec >=2.7.1
```

```suggestion
    , hspec >=2.6.1
```
If you don't need the extra changes in version `2.7.1` this version is included in `lts-13.25`.